### PR TITLE
Remove quotes from genomicIndexEtl parameter

### PR DESCRIPTION
**Pull request by haynescd**

### CODE REVIEW

File: deployment/terraform/step-functions/etl.json.tmpl [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "emrInstanceProfile": "${emr_instance_profile}",
         "emrServiceRole": "${emr_service_role}",
         "esEndpoint":     "${elastic_search_endpoint}",
-        "genomicIndexEtl": "${run_genomic_index_etl}"
+        "genomicIndexEtl": ${run_genomic_index_etl}
```
Comment (evans-g-crsj) [inline deployment/terraform/step-functions/etl.json.tmpl:17]: The other ones are fine too?

Comment (haynescd) [reply deployment/terraform/step-functions/etl.json.tmpl:17]: the others are actual strings

Comment (evans-g-crsj) [reply deployment/terraform/step-functions/etl.json.tmpl:17]: I guess the bool-ish value gets converted to a genuine Python boolean?

Comment (haynescd) [reply deployment/terraform/step-functions/etl.json.tmpl:17]: yea, here is the commit hash from the bug 39a8d607dbd4952a32a853fb6a12b5f84cb0bae7

**Pull request merged**